client/src/app/site/mediafiles/models/view-mediafile.ts

```diff
@@ -11,6 +11,7 @@ import { Searchable } from 'app/site/base/searchable';
 import { ViewGroup } from 'app/site/users/models/view-group';
 
 export const IMAGE_MIMETYPES = [`image/png`, `image/jpeg`, `image/gif`];
+export const IMAGE_USABLE_FOR_PDF_MIMETYPES = [`image/png`, `image/jpeg`];
```

Comment:
I think, this is a too specific declaration, see the `MotionPdfService` for hints